tests/tokenRegistration.test.ts

```diff
@@ -0,0 +1,32 @@
+import TokenRegistrationPageModel from "./TokenRegistrationPageModel";
+import {Selector} from "testcafe";
```

Comment:
Why don't we use Jest?  AFAIK, Jest is default test framework in Mojaloop

Reply:
Testcafe is the test framework that was recommended to me to use for testing this.